radis/db/molparam.py

```diff
@@ -192,7 +192,7 @@ def get(self, M, I, key):
             molecule id
         I: int
             molecule isotope #
-        key: ``'abundance'``, ``'mol_mass'``
+        key: ``'abundance'``, ``'mol_mass'``, ``'isotope_name_exomol'``, 
```

Comment:
```suggestion
        key: ``'abundance'``, ``'mol_mass'``, ``'isotope_name'``, ``'isotope_name_exomol'``, 
```

I also added this one